src/clarity.ts

```diff
@@ -8,6 +8,15 @@ export { version } from "./core";
 export function start(customConfig?: IConfig) {
   if (state !== State.Activated) {
     mapProperties(customConfig, null, true, config);
+    const uploadUrl = config.uploadUrl;
+    if(uploadUrl) {
+      if(!config.urlBlacklist) {
+        config.urlBlacklist = [uploadUrl]
+      } else if (config.urlBlacklist.indexOf(uploadUrl) === -1){
+          config.urlBlacklist.push(uploadUrl);
+      }
+    }
+    
```

Comment:
This isn't a right place to take care of this. Take a look at performance.ts: 50